### PR TITLE
cleanup | models: Remove unnecessary schema properties

### PR DESCRIPTION
**Pull request by cihangir**

### CODE REVIEW

File: workers/social/lib/social/models/account.coffee [hunk 1]
```diff
@@ -178,29 +178,7 @@ module.exports = class JAccount extends jraphical.Module
...
-      shareLocation         : Boolean
       socialApiId           : String
-      skillTags             : [String]
-      locationTags          : [String]
-      systemInfo            :
-        defaultToLastUsedEnvironment :
-          type              : Boolean
-          default           : yes
-      # counts                : Followable.schema.counts
-      counts                :
```
Comment (gokmen) [inline workers/social/lib/social/models/account.coffee]: we had some place where we were updating counts on each login/load afair.

Comment (gokmen) [inline workers/social/lib/social/models/account.coffee]: we may need to remove that part as well.

**Review by gokmen — APPROVED** [review]

File: workers/social/lib/social/models/account.coffee (hunk 1 above)
Comment (cihangir) [inline workers/social/lib/social/models/account.coffee]: JGroup has them, it seems we've removed for users previously

**Comment by cihangir** [comment]
PTAL

**Review by gokmen — APPROVED** [review]
👍

**Pull request merged**